authentication/src/test/java/io/camunda/authentication/session/WebSessionRepositoryTest.java

```diff
@@ -99,6 +99,21 @@ void deleteById() {
     assertThat(webSessionRepository.findById(webSession.getId())).isNull();
   }
 
+  @Test
```

Comment:
💭 FYI, we have a `RegressionTest` annotation, which I find helpful to explain the motivation behind some of the tests or decisions which might look not obvious (e.g. using a string vs object). Typical usage is 

```java
@RegressionTest("https://github.com/camunda/camunda/pull/36919")
void shouldBlaBla() {
  // ...
}
```

The link would be usually the issue link, or a PR link if there is no issue.